README.md

```diff
@@ -1,7 +1,7 @@
-# opx-linux-config
+## opx-linux-config
```

Comment:
Second level header for the primary header of the document? I can't see any reasons for this change.